test/unit/service_test.rb

```diff
@@ -530,7 +530,7 @@ class AsynchronousDeletionOfService < ActiveSupport::TestCase
     test 'updating state without using state machine' do
       service = FactoryGirl.create(:simple_service)
       service.stubs(last_accessible?: false)
-      DeletePlainObjectWorker.expects(:perform_later).never
+      DeleteObjectHierarchyWorker.expects(:perform_later).never
       System::ErrorReporting.expects(:report_error).once
```

Comment:
I know it is not in this PR but it would be good to know what kind of error it is expecting

Reply:
It's in https://github.com/3scale/porta/pull/6
And the code (from master) of the error report:
https://github.com/3scale/porta/blob/352aa1d31106589b62534567ace5dd2413a376d8/app/models/service.rb#L495